### ISSUE TITLE
Support Django 2.0

### ISSUE THREAD

**Issue by chocoelho** [fixmydjango/fixmydjango-lib#17]
Django 2.0 is not yet supported on fixmydjango.

```
  File "/home/ubuntu/.local/share/virtualenvs/testproject-E_l_Ra6e/lib/python3.6/site-packages/fixmydjango/__init__.py", line 22, in <module>
    original_TECHNICAL_500_TEMPLATE = debug.TECHNICAL_500_TEMPLATE
AttributeError: module 'django.views.debug' has no attribute 'TECHNICAL_500_TEMPLATE'
```

**Comment by iyanuashiri** [comment]
I would love to work on this issue. Can someone guide me in implementing a solution?

**Comment by aericson** [comment]
You can start by adding Django 2.0 to tox. Then run the tests and fixing the errors it shows.

This specific error @chocoelho mentioned is related to a change in Django. You can get some hints about the problem by looking the difference between (Django 1.11 and 2.x):

https://github.com/django/django/blob/stable/1.11.x/django/views/debug.py#L324-L328
and
https://github.com/django/django/blob/master/django/views/debug.py#L329-L334

I'm not sure what other issues you will face.

**Comment by andersonresende** [comment]
@iyanuashiri are you working on it?

**Comment by iyanuashiri** [comment]
Yes. I am sorry it is taking time to complete. I have been busy. But will finish this week. Thank you
